test_async/consumer/dune

```diff
@@ -0,0 +1,4 @@
+(executable
+  (name consumer)
```

Comment:
I would also add

```
  (public_name kafka-async-consumer)
  (package kafka_async)
```

because it's convenient to have command-line tools even if redundant with those provided by Kafka.